### PR TITLE
Pagination Fix

### PR DESCRIPTION
**Pull request by LTA-Thinking**
## Description
See #4438 

## Related issues
Addresses [AB#124924](https://microsofthealth.visualstudio.com/f8da5110-49b1-4e9f-9022-2f58b6124ff9/_workitems/edit/124924)

## Testing
Describe how this change was tested.

## FHIR Team Checklist
- **Update the title** of the PR to be succinct and less than 65 characters
- **Add a milestone** to the PR for the sprint that it is merged (i.e. add S47)
- Tag the PR with the type of update: **Bug**, **Build**, **Dependencies**, **Enhancement**, **New-Feature** or **Documentation**
- Tag the PR with **Open source**, **Azure API for FHIR** (CosmosDB or common code) or **Azure Healthcare APIs** (SQL or common code) to specify where this change is intended to be released.
- Tag the PR with **Schema Version backward compatible** or **Schema Version backward incompatible** or **Schema Version unchanged** if this adds or updates Sql script which is/is not backward compatible with the code.
- [ ] CI is green before merge [Build Status](https://microsofthealthoss.visualstudio.com/FhirServer/_build/latest?definitionId=27&branchName=main) 
- Review [squash-merge requirements](https://github.com/microsoft/fhir-server/blob/master/SquashMergeRequirements.md)

### Semver Change ([docs](https://github.com/microsoft/fhir-server/blob/master/docs/Versioning.md))
Patch|Skip|Feature|Breaking (reason)

### CODE REVIEW

**Review by fhibf — APPROVED** [review]

File: samples/templates/default-azuredeploy-docker.json [hunk 1]
```diff
@@ -720,7 +720,7 @@
                 "tier": "Basic"
             },
             "properties": {
-                "adminUserEnabled": "true"
+                "adminUserEnabled": "false"
```
Comment (fhibf) [inline samples/templates/default-azuredeploy-docker.json:723]: Is this change intentional?

Comment (LTA-Thinking) [reply samples/templates/default-azuredeploy-docker.json:723]: Yes, it is already that value in main.
